examples/meta/src/base_api/put_get.sg

```diff
@@ -1,5 +1,5 @@
 KNN knn()
-knn.put("m_k", 2)
+knn.put("k", 2)
```

Comment:
this will break integration tests, so you will need to update the test data file.
See the developing.md readme for details

Reply:
If it doesnt break the integration tests, then no change is needed ...

Reply:
actually nevermind. as long as you change all the example code listings, this is not a problem